CHANGES.md

```diff
@@ -148,6 +148,10 @@
  - Give `Goals` panel focus back if it has lost it (in case of
    multiple panels in the second viewColumn of Vscode) whenever
    user navigates proofs (@Alidra @ejgallego, #722, #725)
+   code points for (Léo Stefanesco, @ejgallego, #624, fixes #620,
```

Comment:
What happened here?

Reply:
Was tweaking the wording of a previous entry and messed up the rebase, Thanks a lot!